entity-view/testsuite/src/test/java/com/blazebit/persistence/view/testsuite/correlation/general/model/DocumentCorrelationViewSubselect.java

```diff
@@ -32,39 +30,22 @@
  * @since 1.2.0
  */
 @EntityView(Document.class)
-public abstract class DocumentCorrelationViewSubselect implements DocumentCorrelationView {//extends DocumentCorrelationView {
```

Comment:
No, the removal was on purpose. This is not needed anymore. I just had it in there so I could "turn off" some correlations for testing specific ones. If I ever have the need of doing that again, I will create a separate model for testing.